staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -75,14 +79,14 @@ type Event struct {
 	// API object from the request, in JSON format. The RequestObject is recorded as-is in the request
 	// (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
-	// Omitted for non-resource requests.  Only logged at RequestObject Level and higher.
+	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestBody string
+	RequestObject runtime.Unknown
```

Comment:
> I would suggest to use runtime.Object in the internal type and have a serialization to string during conversion to the external type. @deads2k wdyt?

I think I'll leave the exact mechanism to be the easiest available, but I wanted to make certain it wasn't a string.  I think that `runtime.Unknown` will be guaranteed to pass straight through without anything trying to encode or decode directly.  If we use a `runtime.Object` it will be affected by the outer encoding structure (again, I think).  `runtime.RawExtension` may work internal and external as well.

I really just wanted to guarantee this didn't end up as a string so that we'd be able to directly inspect with a jsonpath filter or the like against incoming data if we wished without having to pull out the information.  Someday jsonpath may catch up to xml and we'll get streaming evaluators.